Cartesian_kernel/include/CGAL/Cartesian/Scaling_rep_3.h

```diff
@@ -59,6 +60,12 @@ class Scaling_repC3 : public Aff_transformation_rep_baseC3<R>
     return d;
   }
 
+  virtual Plane_3  transform(const Plane_3 &p) const
+  {
+    // direction ( which is (p.a(), p.b(), p.c())) does not change
+    return Plane_3(p.a(),p.b(),p.c(), p.d()*scalefactor_);
```

Comment:
As an aside, would it work if `scalefactor_` was a Vector_3 for non-uniform scaling?

Reply:
I would say that when you non-uniform scale you change the normal of a plane.
![image](https://user-images.githubusercontent.com/3263539/115706568-150f8200-a36e-11eb-8914-1e9e764359b3.png)

Reply:
Ah, thanks for the explanation, and picture.

Reply:
I think we could do special treatment for mirroring and or 90 degree rotations, but for now I stop with the affine transformations.  We should return to the hash map, where I think that it's your turn to do the next steps (back).